extensions/undertow/runtime/src/main/java/io/quarkus/undertow/runtime/UndertowDeploymentRecorder.java

```diff
@@ -658,9 +658,9 @@ public ByteBuf allocateBuffer() {
         @Override
         public ByteBuf allocateBuffer(boolean direct) {
             if (direct) {
-                return PartialPooledByteBufAllocator.DEFAULT.directBuffer(defaultBufferSize);
+                return ByteBufAllocator.DEFAULT.directBuffer(defaultBufferSize);
             } else {
-                return PartialPooledByteBufAllocator.DEFAULT.heapBuffer(defaultBufferSize);
+                return ByteBufAllocator.DEFAULT.heapBuffer(defaultBufferSize);
```

Comment:
What's the reason of this change?

Reply:
https://sonarcloud.io/project/issues?id=gwenneg_quarkus&open=AXAxfMfPgbHBif7sECGm&resolutions=FIXED&severities=CRITICAL

The `DEFAULT` constant is actually defined in `ByteBufAllocator` which is implemented by `PartialPooledByteBufAllocator`.